### PR TITLE
First draft of travel tips (sans hotels + buses)

### PR DESCRIPTION
**Pull request by devjack**
Covers off:
* Airport arrivals (MEL and acknowledges AVV)
* Skybus
* MyKi
* Trams & Trains
* Routes to venue (incl. tips for ride share)

### CODE REVIEW

File: src/content/pages/attend/travel.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Interstate & International travel
+Book travel to Melbourne Tullamarine Airport (MEL). Melbourne also has a second airport, Avalon Airport (AVV), which is significantly further from the city and less well-served by public transport.
+
+Both MEL and AVV are served by [SkyBus](https://www.skybus.com.au/), a shuttle bus service that runs regularly between airports and the city. Tickets can be purchased upon arrival but are cheaper ($40 return for an adult) if pre-booked online.
+
+## Public transport
+Melbourne has a single ticketing system for all public transport, including trains, trams, and buses. The system is called [Myki](https://www.ptv.vic.gov.au/tickets/Myki). You can purchase a Myki card at the airport, most train stations, and many convenience stores around town. The card costs $6 and needs to be topped up with money for travel. 
+
+### Trams
+Melbourne has an extensive and easy to use tram network. Travel is free inside [the green free tramzone](https://www.ptv.vic.gov.au/assets/PDFs/Maps/Network-maps/5b4a3efe80/PTV-Free-Tram-Zone-Map.pdf) covering most of the CBD, but you need to 'tap on' with your Myki if travelling outside hte green zone.
```
Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:20]: i feel like `tramzone` should be two separate words?

Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:20]: `tramzone` should probs be `tram-zone`

`hte` to `the

File: src/content/pages/attend/travel.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+### Trains
+Train fares in Melbourne also use the Myki system. The closest train station to the conference venue is Southern Cross Station, which is a major hub for regional and interstate trains. Note that regional and interstate trains are not covered by the Myki system and are ticketed seperately.
+
+# Arriving at PyCon AU 2024
+This year's conference is held in the Goldfields Theatre at the Melbourne Convention and Exhibition Centre (MCEC). The MCEC is located at 1 Convention Centre Place, South Wharf, Melbourne, Victoria, 3006.
```
Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:26]: replace `The MCEC is` with `MCEC is`? sounds better that way and other sentences drop the `the`.

File: src/content/pages/attend/travel.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+MCEC is a large venue with multiple entrances. The Goldfields Theatre is located on the ground floor, accessible from the main entrance on Convention Centre Place. The venue is fully accessible, with ramps and elevators available for those who need them.
+
+We recommend the following routes when arriving via:
```
Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:30]: rephrase to "We recommend the following routes when arriving..." or "We recommend the following routes when travelling by the following transport methods:" to remove the redundant 'via'.

File: src/content/pages/attend/travel.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## via Tram
+If arriving on routes 35,70 and 75, alight at South Wharf/Wurundjeri Way and its approx 10m walk across Seafarers Bridge.
```
Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:33]: Split into two sentences and add a space between route numbers: 
"If arriving on routes 35, 70 and 75, alight at South Wharf/Wurundjeri Way. From there, the venue is a 10 minute walk across Seafarers Bridge."

(avoid using just `10m` as i was confused on if that was 'minutes' or 'metres').

Comment (libbyberrie) [reply src/content/pages/attend/travel.mdx:33]: Add afterwards "While this route involves a longer walk, it stops within the free tram zone, so you won't need to pay for transportation."

File: src/content/pages/attend/travel.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+If arriving on routes 12, 96, 109, alight at Stop 124A Casino/MCEC/Clarendon St (Southbank) and walk via MCEC Exhibition halls to the far end.
```
Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:35]: "Please note: this tram stop ends outside the free tram zone. You will need to tap off your Myki upon arrival to the event if you are travelling from within that zone, and pay the standard fare."

File: src/content/pages/attend/travel.mdx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+Note: The Clarendon St tram stop is outside the free tram zone, and some delegates may enjoy the additional stroll across the bridge for free tram travel to the conference.
```
Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:37]: i think this is better split into two sentences next to each prospective route. See comments above.

File: src/content/pages/attend/travel.mdx [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+## via Train
+If arriving by train, alight at Southern Cross Station and walk across the Yarra River via the Seafarers Bridge. Alternatively change to tram routes 12, 96 or 109; see above for details.
```
Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:40]: Comma between `Alternatively, change`

File: src/content/pages/attend/travel.mdx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+## via Car or Ride share
+Instead of using the conference venue address, use google maps to route your drop off to the Pan Pacific Melbourne (hotel), The LUME Melbourne, or DFO. The Goldfields foyer has an entrance onto the same drop off / turning circle and it is well suited for pickup/drop-off.
+
+If you use MCEC's address via Google Maps, you'll be able to enjoy the scenic river-side walk past the exhibition halls (along side delegates ariving by tram).
```
Comment (libbyberrie) [inline src/content/pages/attend/travel.mdx:45]: Side delegates is a little vague. "Other attendees" is more accessible here.

**Review by libbyberrie — CHANGES_REQUESTED** [review]
Just some typo fixes and small nitpics here :)

File: src/content/pages/attend/travel.mdx [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+## Public transport
+Melbourne has a single ticketing system for all public transport, including trains, trams, and buses. The system is called [Myki](https://www.ptv.vic.gov.au/tickets/Myki). You can purchase a Myki card at the airport, most train stations, and many convenience stores around town. The card costs $6 and needs to be topped up with money for travel. 
+
+### Trams
+Melbourne has an extensive and easy to use tram network. Travel is free inside [the green free tramzone](https://www.ptv.vic.gov.au/assets/PDFs/Maps/Network-maps/5b4a3efe80/PTV-Free-Tram-Zone-Map.pdf) covering most of the CBD, but you need to 'tap on' with your Myki if travelling outside hte green zone.
+
+### Trains
+Train fares in Melbourne also use the Myki system. The closest train station to the conference venue is Southern Cross Station, which is a major hub for regional and interstate trains. Note that regional and interstate trains are not covered by the Myki system and are ticketed seperately.
```
Comment (urcheraus) [inline src/content/pages/attend/travel.mdx:23]: > Note that regional and interstate trains are not covered by the Myki system and are ticketed seperately.

No longer accurate, Myki now works for
> V/Line trains between metropolitan Melbourne and Eaglehawk/Epsom, Seymour, Traralgon, Waurn Ponds and Wendouree

as per https://www.ptv.vic.gov.au/tickets/Myki

File: src/content/pages/attend/travel.mdx (hunk 8 above)
Comment (daisylb) [reply src/content/pages/attend/travel.mdx:45]: I think it was originally intended to be "alongside delegates"?

File: src/content/pages/attend/travel.mdx (hunk 9 above)
Comment (daisylb) [reply src/content/pages/attend/travel.mdx:23]: Both statements are half true; some V/Line services still use separate paper tickets, the Melbourne-Sydney XPT is ticketed separately by NSW TrainLink, and the Melbourne-Adelaide Overland is ticketed by Journey Beyond.

File: src/content/pages/attend/travel.mdx (hunk 1 above)
Comment (daisylb) [reply src/content/pages/attend/travel.mdx:20]: Also worth mentioning the venue itself is not within the FTZ

**Pull request merged**